src/app/admin/_components/faq/Edit.js

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react';
+import {
+  Edit,
+  SimpleForm,
+  TextInput,
+  BooleanInput,
+  NumberInput,
+  Toolbar as BaseToolbar,
+  SaveButton,
+  required,
+  useNotify,
+  useRecordContext,
+  useDelete,
+  Confirm,
+  Button,
+} from 'react-admin';
+import { useActiveFaqs } from './hooks';
+import { AnswerTextInput } from './AnswerTextInput';
+import { MAX_ACTIVE_FAQS, MIN_ACTIVE_FAQS } from './consts';
+
+const TOO_MANY_ACTIVE_FAQS = `Too many active FAQs. Please deactivate some first.`;
+const TOO_FEW_ACTIVE_FAQS = `At least five FAQ must be active. Pleace activate some first.`;
+
+function DeleteButton() {
+  const record = useRecordContext();
+  const notify = useNotify();
+  const { total: activeFaqsCount } = useActiveFaqs();
+  const [open, setOpen] = useState(false);
+
+  const [remove, { isLoading }] = useDelete('faq', { id: record && record.id });
```

Comment:
Can't it be replaced to `{ id: record?.id }`

Reply:
it can, will do